.nuspec/Microsoft.Maui.Controls.SingleProject.targets

```diff
@@ -68,6 +68,44 @@
     <TizenSharedPrefix>$(TizenProjectFolder)shared</TizenSharedPrefix>
   </PropertyGroup>
 
+  <!-- Add metadata indicating that the platform-specific files are not part of every build configuration. -->
+  <ItemGroup Condition=" '$(SingleProject)' == 'true' ">
```

Comment:
If each `<ItemGroup>` here checks `$(SingleProject)`, could we put all of them under a single `<ItemGroup>` element? Then the `Condition` would be on each `<Compile>` item.

Reply:
Done.